src/olympia/addons/tests/test_models.py

```diff
@@ -1919,14 +1919,31 @@ class TestAddonDependencies(TestCase):
     def test_dependencies(self):
         ids = [3615, 3723, 4664, 6704]
         addon = Addon.objects.get(id=5299)
+        dependents = Addon.objects.in_bulk(ids)
```

Comment:
regarding the naming, shouldn't it all be `dependencies` and `dependency` instead of `dependent` and `dependents`? Maybe I'm just wrong so feel free to ignore that.